hash/zx81_cass.xml

```diff
@@ -1276,6 +1333,20 @@ Known dumps not yet added (as of 2023-06-14):
 		</part>
 	</software>
 
+	<software name="chateauh">
+		<description>Le Chateau Hante</description>
+		<year>1984</year>
+		<publisher>Sprites</publisher>
+		<info name="alt_title" value="Le Chateau Hanté" /> <!-- in game title -->
+		<info name="language" value="French" />
```

Comment:
Is there a good reason to have “Hante” without the acute accent in the description?  It looks like it’s just a result of the printing system they used lacking support for accented uppercase letters, not an actual stylistic choice.

Reply:
I changed the description, but what is the rule? I thought it would always be the name that would be in the box/inlay, regardless of not having the accents.
Because in this case there's a lot of games (ZX Spectrum and Oric 1) that the accented letters are missing.

Reply:
Well you converted it to title case, which isn’t a verbatim copy of what’s shown on the box anyway.  The only reason it has no accent on the box is because it’s in all uppercase and their printing system doesn’t appear to support accented uppercase.  When it isn’t forced into uppercase without accents it’s correctly written as “Hanté”.